docs/phylo_bgs.slim

```diff
@@ -0,0 +1,34 @@
+// The following constants need to be defined:
+// - outfile: path to save the tree sequence of the simulation.
+// - popsize: population size.
+// - num_gens: run simulation for num_gens generations.
+// - infile: a string with path to tree sequence to start from; optional.
+initialize()
+{
+    initializeSLiMModelType("WF");
+    initializeTreeSeq();
+    initializeMutationRate(1e-6);
+    initializeMutationType("m1", 0.5, "f", -0.01);
+    initializeGenomicElementType("g1", m1, 0.1);
+    initializeGenomicElement(g1, 0, 1e4-1);
+    initializeRecombinationRate(1e-8);
+}
+
+1 early() {
+    // if no input tree sequence is provided, then start a subpopulation
+    if (infile == "") {
```

Comment:
alternatively, this could be
```
if (!exists("infile")) {
```
allowing the infile to be skipped entirely.

Reply:
then it is trickier to deal with this in the print of the slim command.